### PR TITLE
Add recap blog post for Contributor Summit Barcelona 2019

### PR DESCRIPTION
**Pull request by jonasrosland**
This will add a recap blog post of Contributor Summit Barcelona.

Signed-off-by: jonasrosland <jrosland@vmware.com>

### CODE REVIEW

File: content/en/blog/_posts/2019-06-25-recap-of-contributor-summit-bcn-2019.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Contributor Summit in Barcelona kicked off KubeCon + CloudNativeCon in a big way as it was the **largest contributor summit** to date with 331 people signed up, and only 9 didn't pick up their badges!
+
+## Contributor Celebration
+
+Sunday evening before the main event we held a **Contributor Celebration**, which was very well attended. We hope that all new and current contributors felt welcome and enjoyed the food, the music, and the company.
+
+![contributor-celebration2](https://live.staticflickr.com/65535/46981485515_561bb324b2_z.jpg)
+![contributor-celebration](https://live.staticflickr.com/65535/46981484655_8122564557_z.jpg)
+
+## New Contributor Workshops
```
Comment (castrojo) [inline content/en/blog/_posts/2019-06-25-recap-of-contributor-summit-bcn-2019.md]: This wouldn't be a bad place to cross link the playlist on youtube so people who didn't attend can check the content out: https://www.youtube.com/watch?list=PL69nYSiGNLP2WTJ6P8sQenhf0RY-JqF5L&v=BQ7y2TFOzF4

**Comment by castrojo** [comment]
/lgtm 

I think adding a link to the youtube list would be great, but not a blocker.

File: content/en/blog/_posts/2019-06-25-recap-of-contributor-summit-bcn-2019.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+At the end of the summit, both new and current contributors had a chance to sit down with SIG chairs and members. The goal of this was to make sure that contributors got to know even more individuals in the project, hear what some of the SIGs actually do, and sign up to be a part of them and learn more.
+
+![img](https://live.staticflickr.com/65535/47108248464_97eb2bbbb6_k.jpg)
+![img](https://live.staticflickr.com/65535/47845452032_a3d478beb9_k.jpg)
+
+
+## Join us!
+
+Interested in attending the Contributor Summit in San Diego? [You can get more information here](https://events.linuxfoundation.org/events/contributor-summit-north-america-2019/), sign up and we will notify you when registration opens.
```
Comment (geekygirldawn) [inline content/en/blog/_posts/2019-06-25-recap-of-contributor-summit-bcn-2019.md]: This is a total nit, but I try never to link the word "here". I usually either just leave it off or change it to say something like "get more information on our event page".

**Comment by jonasrosland** [comment]
@castrojo Added the YouTube playlist, thanks!
@geekygirldawn Fixed your minor nit, thank you!

**Comment by parispittman** [comment]
looks super great; enjoyed the pics!

/lgtm

**Pull request merged**